src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -238,26 +230,41 @@
         {{ profiler_dump_log(log.message) }}
     {% else %}
         {{ profiler_dump_log(log.message, log.context) }}
+    {% endif %}
 
-        <div class="text-small font-normal">
+    <div class="log-metadata">
+        {% if log.channel %}
+            <span class="badge">{{ log.channel }}</span>
+        {% endif %}
+
+        {% if log.errorCount is defined and log.errorCount > 1 %}
+            <span class="log-num-occurrences">{{ log.errorCount }} times</span>
+        {% endif %}
+
+        {% if has_context %}
             {% set context_id = 'context-' ~ category ~ '-' ~ log_index %}
-            <a class="btn btn-link text-small sf-toggle" data-toggle-selector="#{{ context_id }}" data-toggle-alt-content="Hide context">Show context</a>
+            <span><a class="btn btn-link text-small sf-toggle" data-toggle-selector="#{{ context_id }}" data-toggle-alt-content="Hide context">Show context</a></span>
+        {% endif %}
 
-            {% if has_trace %}
-                &nbsp;&nbsp;
-                {% set trace_id = 'trace-' ~ category ~ '-' ~ log_index %}
-                <a class="btn btn-link text-small sf-toggle" data-toggle-selector="#{{ trace_id }}" data-toggle-alt-content="Hide trace">Show trace</a>
-            {% endif %}
-        </div>
+        {% if has_trace %}
+            {% set trace_id = 'trace-' ~ category ~ '-' ~ log_index %}
+            <span><a class="btn btn-link text-small sf-toggle" data-toggle-selector="#{{ trace_id }}" data-toggle-alt-content="Hide trace">Show trace</a></span>
```

Comment:
using  a`<button type=button>` would be better than a `<a>` from an accessibility point of view (btw, a `<a>` without `href` is not even focusable)